tests/typ/lint/scripted-warnings.typ

```diff
@@ -0,0 +1,69 @@
+/// Test scripted warnings (Inspired by https://github.com/typst/typst/issues/1276#issuecomment-1560091418).
+// Ref: false
+
+---
+#set heading(numbering: "1.")
+
+#let myref(label) = locate(loc => {
+    if query(label,loc).len() != 0 {
+        ref(label)
+    } else {
+// Warning: 14-61 could not find reference <test>

```

Comment:
the warning comment can also be indented (at least I think so, I might be wrong about this)

Reply:
that worked, yes